src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue23563.cs

```diff
@@ -0,0 +1,25 @@
+﻿#if ANDROID
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue23563 : _IssuesUITest
+	{
+		public Issue23563(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "Unable to set toolbar overflow menu color if not using shell";
+
+		[Test]
+		[Category(UITestCategories.Navigation)]
+		public void OverflowIconShouldBeRed()
+		{
+			App.WaitForElement("Label");
+			VerifyScreenshot();
```

Comment:
Snapshots available in the latest build.
![image](https://github.com/user-attachments/assets/1034577a-30a2-4d8d-bc61-2343ceaa96e4)

Could you commit the images?

Reply:
Done :)